src/overlays/actors/ovl_Bg_Gate_Shutter/z_bg_gate_shutter.c

```diff
@@ -22,19 +33,109 @@ const ActorInit Bg_Gate_Shutter_InitVars =
     (ActorFunc)BgGateShutter_Update,
     (ActorFunc)BgGateShutter_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gate_Shutter/BgGateShutter_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Gate_Shutter/BgGateShutter_Destroy.s")
+extern UNK_PTR D_06001CD0;
+extern UNK_PTR D_06001DA8;
```

Comment:
I would make those UNK_TYPE since they aren't pointers but rather arrays/structs that may not be identified yet.

Reply:
Gotcha.